specification/purview/data-plane/Azure.Analytics.Purview.Account/readme.md

```diff
@@ -38,3 +46,104 @@ These settings apply only when `--tag=package-2019-11-01-preview` is specified o
 input-file:
 - preview/2019-11-01-preview/account.json
 ```
+
+---
+# Code Generation
+
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-go
```

Comment:
You'd need to remove this too in order to skip building Go.

Reply:
removed it from here and added in readme.go.md

Reply:
This still needs to be removed to skip building Go.